docs/projects/ad_fmclidar1_ebz/index.rst

```diff
@@ -0,0 +1,260 @@
+.. _ad_fmclidar1_ebz:
+
+AD-FMCLIDAR1-EBZ HDL project (OBSOLETE)
+================================================================================
+
+.. warning::
+
+   The support for :git-hdl:`AD-FMCLIDAR1-EBZ <hdl_2019_r1:projects/ad_fmclidar1_ebz>`
+   HDL project has been discontinued, the latest release branch where it can be found
+   is ``hdl_2019_r1``. This page is left for legacy purposes only.
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD-FMCLIDAR1-EBZ <AD-FMCLIDAR1-EBZ>` is a prototyping platform for
+LiDAR applications that can be used on FPGA development boards enabled with FMC
+HPC connector and JESD204B support capability. It offers developers a
+working-out-of-box platform that can be used for developing software and
+algorithms for a broad range of applications.
+
+The full system hardware includes Data Acquisition(DAQ) Board, Laser Transmitter
+Board, and the AFE Receiver Board.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+- :adi:`AD-FMCLIDAR1-EBZ <AD-FMCLIDAR1-EBZ>`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+- :xilinx: `ZC706 <https://www.xilinx.com/products/boards-and-kits/ek-z7-zc706-g.html>` on FMC HPC
+- :xilinx: `ZCU102 <https://www.xilinx.com/products/boards-and-kits/ek-u1-zcu102-es2-g.html>` on on FMC HPC0
+- :intel: `A10SoC <https://www.altera.com/products/boards_and_kits/dev-kits/altera/arria-10-soc-development-kit.html>` on FMC HPC
+
+The Arria10SOC carrier requires a hardware rework to function correctly. The rework
+connects FMC_A header pins directly to the FPGA so that they can be accessed by the fabric.
+
+Changes required:
+
+REMOVE: R575, R576, R621, R633, R612, R613
+
+POPULATE: R574, R577, R620, R632, R610, R611
+
+Block design
```

Comment:
The CPU interrupts, GPIO and SPI sections should be added as well.
The GPIOs which should be added, for example, are these ones: https://github.com/analogdevicesinc/hdl/blob/hdl_2019_r2/projects/ad_fmclidar1_ebz/zcu102/system_top.v#L167-L173.